src/System.Reflection.Metadata/tests/Metadata/Ecma335/Encoding/BlobEncodersTests.cs

```diff
@@ -388,7 +388,350 @@ public void FixedArgumentsEncoder_AddArgument()
         // NamedArgumentTypeEncoder
         // CustomAttributeArrayTypeEncoder
         // CustomAttributeElementTypeEncoder
-        // SignatureTypeEncoder
+
+        [Fact]
+        public void SignatureTypeEncoder_Primitives()
+        {
+            var b = new BlobBuilder();
+            var e = new SignatureTypeEncoder(b);
+            Assert.Same(b, e.Builder);
+
+            e.Boolean();
+            AssertEx.Equal(new byte[] { 0x02 }, b.ToArray());
```

Comment:
Separate, but why do we have AssertEx.Equal? I believe Assert.Equal from xunit does what we want here.